go.mod

```diff
@@ -70,25 +70,25 @@ require (
 	go.etcd.io/etcd/client/pkg/v3 v3.5.10
 	go.etcd.io/etcd/client/v3 v3.5.10
 	go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful v0.42.0
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.42.0
-	go.opentelemetry.io/otel v1.19.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0
-	go.opentelemetry.io/otel/sdk v1.19.0
-	go.opentelemetry.io/otel/trace v1.19.0
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.46.0
+	go.opentelemetry.io/otel v1.20.0
```

Comment:
looking at the changelog,

[1.20.0/0.43.0](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.20.0) 2023-11-10
This release brings a breaking change for custom trace API implementations. Some interfaces (TracerProvider, Tracer, Span) now embed the go.opentelemetry.io/otel/trace/embedded types. Implementors need to update their implementations based on what they want the default behavior to be. See the "API Implementations" section of the [trace API](https://pkg.go.dev/go.opentelemetry.io/otel/trace) package documentation for more information about how to accomplish this.

Does that apply to us?

Reply:
It does not.  We do not (currently) have any custom implementations of the tracing interfaces in-tree.  It would cause a compilation failure if we were impacted.